test/erlfmt_cli_SUITE.erl

```diff
@@ -236,6 +238,11 @@ exclude_check(Config) when is_list(Config) ->
     ?assertNotMatch(nomatch, string:find(WithoutBroken, "[warn]")),
     ?assertMatch(nomatch, string:find(WithoutBroken, "broken.erl")).
 
+range_check(Config) when is_list(Config) ->
```

Comment:
Nice <3

Is it possible to add one more with partial range?

I guess this would require making a more complicated stdio_test function, hmmm.
Maybe we can do this in a follow up commit, after this one is merged

Reply:
Exactly, that would be a separate commit, or even a separate issue (a new feature, whose benefit/use-case are unclear to me).